### PR TITLE
AQCU-1207: Convert aquarius-domain to use swagger

### PR DESCRIPTION
**Pull request by zmoore-usgs**
Should be merged with: https://github.com/USGS-CIDA/AQCU/pull/773

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -1,93 +1,95 @@
...
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-	<modelVersion>4.0.0</modelVersion>
-	
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
 	<groupId>gov.usgs.cida</groupId>
 	<artifactId>aquarius-domain</artifactId>
-	<version>17.1.96-5-SNAPSHOT</version>
-	
+    <version>17.1.96-5</version>
```
Comment (zmoore-usgs) [inline pom.xml]: Not sure about this version change. If the "-4" is also tied to the AQ version then this should be changed to something such as 17.1.96-4.1 or similar

**Comment by thongsav-usgs** [comment]
Does Swagger have a file format that downloads that we can persist on file like the XSDs? I would like to get away from the build-time dependency of an AQUARIUS server. Does the plugin allow us to point to static files in source?

**Comment by zmoore-usgs** [comment]
You can download the Swagger JSON files manually from the server and those can be fed into the plugin somehow, but I haven't attempted to do it that way so I'm not sure how it works when providing multiple files. 

As I understand it this project should only need to be re-built whenever there is a new AQ version released and we would also need the AQ server up and available in order to manually grab the Swagger JSON files if we went in that direction. I'm not sure having them locally would really provide much benefit with the infrequency at which this project needs to be built?

**Comment by thongsav-usgs** [comment]
I imagine the benefit would be for posterity. Before we started checking the XSDs into source we had no way to look back at old versions of the API and do a "what changed from version x to version y". Additionally we are unable to go back and build previous versions of the API without this. There is an edge case where we have to go back and hot fix an older version of the API (eg: when we make changes to our parsers). I agree it's an edge case, but that was the justification for checking in the XSDs, and I think it applies to swagger JSON as well.

**Comment by zmoore-usgs** [comment]
Ah gotcha, that makes sense to me. I'll look into running the process on local JSON

**Pull request merged**